packages/@aws-cdk/aws-iam/lib/policy-document.ts

```diff
@@ -213,6 +248,10 @@ export class PolicyStatement extends Token {
     return this.addPrincipal(new ArnPrincipal(arn));
   }
 
+  public addArnPrincipal(arn: string): PolicyStatement {
```

Comment:
Would express as `public addArnPrincipal(arn: string): this`.

Reply:
Done (in all methods)